code/modules/species/shadekin/shadekin_blackeyed.dm

```diff
@@ -107,12 +109,155 @@
 		BP_R_FOOT = list("path" = /obj/item/organ/external/foot/right/crewkin),
 	)
 
+	var/list/crew_shadekin_abilities = list(
+		/datum/power/crew_shadekin/crewkin_regenerate_other,
+		/datum/power/crew_shadekin/crewkin_create_shade,
+	)
+
+	var/list/crew_shadekin_ability_datums = list()
+	var/kin_type
+	var/energy_light = 0.25
+	var/energy_dark = 0.75
+
 	unarmed_types = list(
 		/datum/unarmed_attack/stomp,
 		/datum/unarmed_attack/kick,
 		/datum/unarmed_attack/claws,
 		/datum/unarmed_attack/bite/sharp,
 	)
 
-/datum/species/shadekin/can_breathe_water()
+/datum/species/crew_shadekin/New()
```

Comment:
pretty sure we dont do new() anymore, use initilize() instead

Reply:
As discussed in Discord, going to use New()